### ISSUE TITLE
TextField breaks controls

### ISSUE THREAD

**Issue by stx** [RafaelBarbosatec/bonfire#250]
Thanks for working on this wonderful software!

Tested with the repo example in Chrome, this breaks controls:

```
overlayBuilderMap: {
  'chat': (context, game) => Material(child: TextField()),
},
initialActiveOverlays: const [
  'chat',
],
```

If you tap the text field, controls will still work. If you press enter or tap out of the text field, controls stop working.

Weirdly, if you tap _back_ into the text field, controls start working again. Is this a Bonfire issue or Flame issue?

**Comment by RafaelBarbosatec** [comment]
Hi @stx !
Thanks for this report!
I will evaluate this case!

**Comment by stx** [comment]
Found the issue. FlameGame doesn't take focus back when tapping on them. Here's my fix:

```
GestureDetector(
  onTap: gameFocus.requestFocus,
  behavior: HitTestBehavior.translucent,
  child: GameWidget(
    game: engine,
    focusNode: gameFocus,
  ),
),

**Comment by RafaelBarbosatec** [comment]
Thanks so much !
The params `focusNode`, `autofocus` and `mouseCursor` will available in next release.

**Comment by RafaelBarbosatec** [comment]
Available in 2.4.2 version.
